courses-app/src/app/shared/components/course-card/course-card.component.ts

```diff
@@ -0,0 +1,45 @@
+import {Component, Input, OnInit} from '@angular/core';
+import {ICourse} from "../../../../dto";
+import * as moment from "moment";
+import {
+  DATE_FORMAT,
+  CREATED_LABEL_TEXT,
+  DURATION_LABEL_TEXT,
+  AUTHORS_LABEL_TEXT
+} from "../../constants";
+
+const TEXT_MAX_LENGTH: number = 500;
+
+@Component({
+  selector: 'app-course-card',
```

Comment:
it seems that you use it only in courses module, so it make sense to move it there
main idea around shared module - to put here things that you would like to reuse 
https://angular.io/guide/sharing-ngmodules
![image](https://user-images.githubusercontent.com/33813087/118039937-2f3aef80-b371-11eb-82a1-afdba3bb714c.png)